plugins/catalog-backend/src/schema/openapi.yaml

```diff
@@ -82,6 +87,12 @@ components:
         type: array
         items:
           type: string
+      examples:
+        'Get groups':
+          value: ['kind=group']
+        'Get orphaned components':
+          value:
+            ['kind=component', 'metadata.annotations.backstage.io/orphan=true']
```

Comment:
This is actually not right! These examples are really needed and I'm happy you're doing them :)

See [this comment](https://github.com/backstage/backstage/blob/56c4216dbf643505e7a35736b7f80ed51bc15dcc/packages/catalog-client/src/types/api.ts#L47)

## get entities with EITHER kind component, OR being orphaned (no matter what the kind is)

`['kind=component', 'metadata.annotations.backstage.io/orphan=true']`

## get entities that are components AND orphaned

`['kind=component,metadata.annotations.backstage.io/orphan=true']`

Reply:
That's indeed true. Changed it now